UPGRADE.md

```diff
@@ -2,6 +2,23 @@
 
 ## dev-develop
 
+### Content navigation
+
+The interfaces for the content navigation have been changed, so you have to
+apply these changes if you have used a content navigation in your bundle.
+
+Basically you can delete the `NavigationController` delivering the content
+navigation items together with its routes. It's now common to suffix the
+classes providing content navigation items with `ContentNavigationProvider`.
+
+These classes have to implement the `ContentNavigationProviderInterface` and be
+registered as services as described in the
+[documentation](http://docs.sulu.io/en/latest/cookbook/using-the-tab-navigation.html).
```

Comment:
This link won't work before https://github.com/sulu-io/sulu-docs/pull/25/files has been merged.